config/webpack.ts

```diff
@@ -177,7 +177,15 @@ export default (env: Env) => {
           terserOptions: {
             ecma: 2020,
             module: true,
-            compress: { passes: 3, toplevel: true },
+            compress: {
+              passes: 3,
+              toplevel: true,
+              unsafe: true,
+              unsafe_math: true,
+              unsafe_proto: true,
+              pure_getters: true,
+              pure_funcs: ['JSON.parse', 'Object.values', 'Object.keys'],
```

Comment:
Just so I'm understanding this correctly (I can't see us doing this) but if we were relying on `JSON.parse(someString)` to throw an exception if the string wasn't JSON and ignoring the return value, this could change behavior by dropping the call?

Reply:
That's right. I can't really think of where we'd do that though.